.eslintrc.js

```diff
@@ -1,3 +1,20 @@
 module.exports = {
```

Comment:
this is not part of this feature, please remove it and create it in a different branch